Branch xingyu dg contribution (#199)
AY2021S1-CS2103T-T12-1/tp: docs/UserGuide.md

4. Double-click the file to start the app. The similar GUI shown below (Figure 1) should appear in a few seconds. <br>

>    *Note how the app contains some sample data but the installed version on your desktop might have a different data set.*<br>

Reviewer: ```suggestion
   *Note how the app contains some sample data but the installed version on your desktop might have a different data set.*
```

Reviewer: Not sure if this is better? Is the white spacing too big for every picture?

Author: The white spacing is indeed abit too big with 2 <br>
![image](https://user-images.githubusercontent.com/61489581/97188703-c75ef600-17de-11eb-8813-7ea21a7cffc8.png)
added your change! Thanks!

    #<br>![Ui](images/UiWithAnnotation.png) <br>
    Figure 1: Introduction to OneShelf User Interface

5. Type the command `help start` in the command box and press Enter to execute it. <br>
   A new help window shown below (Figure 2) should appear on your desktop.

>     #<br>![Help Window](images/UiHelpStart.png) <br>

Reviewer: The spacing for this seems the best thus far 
![image](https://user-images.githubusercontent.com/46394439/97177438-27e73680-17d1-11eb-940d-2c1ba2aff18c.png)

#### 3.1.2 Adding an item or delivery: `add-i` or `add-d`

Adds inventory item or pending delivery to OneShelf.

##### 3.1.2.a `add-i`

Format: `add-i n/NAME q/QUANTITY [s/SUPPLIER] [max/MAX_QUANTITY] [metric/METRIC] [t/TAG]...​`
* Adds `QUANTITY` from the current quantity of an inventory item if the item already exist.
An item is uniquely identified by its `NAME` and `SUPPLIER`.
* Note 1: `MAX_QUANTITY` denotes the ideal stock level of that particular item.
* Note 2: `TAG` could be used to categorise items. EG: Duck can be tagged as meat. 

> Example 1: `add-i n/DUCK q/10 s/NTUC max/50 metric/KG` using [Figure 1](#bruiimagesuiwithannotationpng-br) as a base.

Reviewer: If i were u i'll add multiple tags too for full set, then example 2 is minimum add = only name and quantity

Author: Got it!

1. Select the Command Box and type in `add-i n/DUCK q/10 s/NTUC max/50 metric/KG`.
<br>

> ![add-i example one step one](images/add-i%20example%20one%20step%20one.PNG)

Reviewer: Forgot to put Figure here

Author: Figure is hyperlinked instead of placed there

<br>
<br>

2. Either press Enter on your keyboard or click Send to execute the command.

3. Success Message will be shown in the Result Display (assuming there isn't an existing same item) and added item will show up in InventoryBook.
<br>

> ![add-i example one step three](images/add-i%20example%20one%20step%20three.PNG)

Reviewer: Another missing figure

<br>
<br>

> Example 2: `add-i n/Tuna q/20 s/NTUC` using [Figure 1](#bruiimagesuiwithannotationpng-br) as a base.

Reviewer: ```suggestion
Example 2: `add-i n/Tuna q/20 s/NTUC` using [Figure 1](#bruiimagesuiwithannotationpng-br) as the starting point.
```

#### 3.2.2 Scrolling through command history

> OneShelf commands are traversable (up to 20 commands) much like Window's command prompt with the arrow up key traversing into previous commands and arrow down key traversing into next commands.

Reviewer: I think can be better rephrased with the action first. By pressing arrow up key ........ you can ........ What do you think?

Author: Paraphrased! Take a look!
